main/main.zkasm

```diff
@@ -56,6 +56,15 @@ setGlobalExitRoot:
 
         %ADDRESS_GLOBAL_EXIT_ROOT_MANAGER_L2 => A
         %SMT_KEY_SC_STORAGE => B
+
+        ; read timestamp given the globalExitRoot
+        $ => D                                  :SLOAD
+
+        ; skip overwrite timestamp if it is different than 0
+        D - 1                                   :JMPN(writeGlobalExitRoot)
```

Comment:
I think his could also be done like this:
-D               :JMPN(skipSetGlobalExitRoot)
$ => D                                  :MLOAD(timestamp)
$ => SR                                 :SSTORE ; Store 'timestamp' in storage position 'keccak256(globalExitRoot, 0)'
skipSetGlobalExitRoot:
....
One label and one JMP less

Reply:
you are right !! I will change it